packages/components/src/Icon/Icon.mdx

```diff
@@ -25,6 +25,27 @@ Use icons as intended. Provided icons have a specific, meaning. To avoid
 confusing users, it’s essential that each image be used in accordance with its
 meaning and recommended usage.
 
+## Design Guidelines
+
+When contributing a new icon, follow these guidelines:
+
+### Size and Weight
+
+- Design within a 20px grid on a 24px artboard
+- Circular icons should have a 20px diameter
+- Lines should have a 2px weight
+
+### Corners
+
+- Right-angle corners should have a 2px radius
+- Irregular corners should have a 1px radius
```

Comment:
Is this true for angles both over and under 90°?

Reply:
Good clarification